File: (8 kyu) String repeat/(8 kyu) String repeat.js
// 1 Plain Solution
function repeatStr (n, s) {
  let str = '';
  
  for (let i = 0; i < n; i++) {
    str = str + s;
  }
  
  return str;
}

// 2 Optimized solution
function repeatStr (n, s) {
  return (
    new Array(n).fill(s)
  ).join('');
}

// 3 Clever solution
function repeatStr (n, s) {
    return s.repeat(n)
}

// 4 Coding golf
repeatStr = (n, s) => s.repeat(n);

File: 8kyu/(8 kyu) String repeat/(8 kyu) String repeat.cs
namespace Solution
{
  public static class Program
  {
    public static string RepeatStr(int n, string s)
    {
      string res = "";
      for (int i = 0; i < n; i++)
      {
        res += s;
      }
      return res;
    }
  }
}
